user_front/locales/ja.json

```
@@ -117,7 +117,8 @@
     "food": "食品申請",
     "saleGoods": "物品販売",
     "purchase": "購入品申請",
-    "placeMessage": "野外・室内どちらかのみの申請でお願いいたします。"
+    "placeMessage": "野外・室内どちらかのみの申請でお願いいたします。",
+    "ItemMessage": "ステージ団体、屋内団体、屋外団体どちらかのみの申請でお願いいたします。"
```

```suggestion
    "ItemMessage": "ステージ団体、屋内団体、屋外団体いずれかのみの申請でお願いいたします。"
```
選択肢が三つなのでどちらはなんか違う感じがして。。。

---

わいともそう思います